Topics/User_Experience/Design_Requirements.md

```diff
@@ -0,0 +1,120 @@
+
+The usability of a product is addressed by the question “Are we designing the thing right?” [1], which considers whether users are able to use the product.
+
+To achieve this goal, design requirements include specifications related to user interfaces, navigation, visual design, and additional elements which contribute to the satisfaction of end users and overall usability. This makes design requirements an integral part of the user experience.
+
+## **Motivation**
+
+When working with partner clients, scoping the problem that clients want to address and understanding the requirements that their ideal solution entails is very important. Establishing design requirements is highly valuable for CSC301 students, who need to create minimum-viable-products (MVPs) from scratch.
+
+### **1. Client Needs** 
+
+After synthesizing user stories, extracting the functional and non-functional design requirements for any design (whether it is a web-application or mobile application) will allow students to gauge the specific features that they will be working on in order to create an MVP that accurately addresses client needs.
+
+### **2. Action Items**
+
+The design requirements make sure that regardless of the platform the solution is being presented in, the main problem the clients present is solved and the user needs are met. They can also be dissected to identify how the requirements match to various front-end, back-end, or database tasks, allowing for students to scope actionable task items.
+
+### **3. Communication**
+
+Accurately defining and documenting requirements makes sure that the communication between team members and clients are not lost and can be referred to later in the project to see if they have been met, especially after usability studies. Benefits include;
+
+- **Clearer communication**: Defining requirements ensures that the developer team and the client are on the same page and they can identify misunderstandings early on.
+- **Asynchronous communication**: With shared understanding and written documentation, certain questions can be addressed asynchronously, to see if a certain feature matches the clients design requirements.
+- **Efficiency**: There is less time spent on actively thinking about what the client had wanted. The written documentation can be easily referred to in later processes and can be used to hold clients accountable as well.
+
+## **Functional Design Requirements**
+
+Functional design requirements specify the goals of the desired product. They focus on the “usefulness” of the product.
+
+Examples include;
+- The system must enable users to verify their accounts using their phone number.
+- The system should support backward and forward navigation in the research article to follow the natural order of sections.
+- The system must allow visitors to sign up for the newsletter by inputting their email. 
+
+### **Relation to User Stories**
```

Comment:
I really like this section of the wiki page. I feel like this may be a common point of confusion for CSC301 students so it is a great way to clarify any misunderstandings about design requirements.